TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot_components/robot/Robot.java

```diff
@@ -37,10 +41,19 @@ public Robot(HardwareMap hardwareMap, Telemetry telemetry) {
 
         spinnerMotor = hardwareMap.get(DcMotor.class, "spinnerMotor");
         intakeMotor = hardwareMap.get(DcMotor.class, "intakeMotor");
-        extenderMotor = hardwareMap.get(DcMotor.class, "extenderMotor");
-        extenderMotor.setMode(DcMotor.RunMode.RUN_TO_POSITION);
+        intakeMotor.setDirection(DcMotorSimple.Direction.REVERSE);
+        extenderMotor = hardwareMap.get(DcMotor.class, "extensionMotor");
+       // extenderMotor.setTargetPosition(0);
+       // extenderMotor.setMode(DcMotor.RunMode.RUN_TO_POSITION);
         dropperServo = hardwareMap.get(Servo.class, "dropperServo");
-        intakeBucketFlipServo = hardwareMap.get(Servo.class, "intakeServo");
+        //intakeBucketFlipServo = hardwareMap.get(Servo.class, "intakeServo");
+
+
+        dropperServo.setPosition(DROPPER_MIN);
+        //intakeBucketFlipServo.setPosition(INTAKE_DWN);
+
+        intakeAngle = INTAKE_DWN;
```

Comment:
again, intake servo is gone and dropper servo needs testing and improvement

Reply:
done